src/sidebar/util/test/oauth-client-test.js

```diff
@@ -229,16 +228,14 @@ describe('sidebar/util/oauth-client', () => {
       });
 
       return authorized.then(() => {
-        const params = {
+        const params = new URLSearchParams({
```

Comment:
`URLSearchParams` supports passing a record to the constructor, but this requires Safari >= 11 which is newer than our current baseline. That's not a problem in tests however. We can probably bump the Safari requirement to version 11+ for all of our projects, but I didn't want to make that a dependency of this work.

Reply:
Yeah, that makes sense.